lib/filter/filter-throttle.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021 Balabit
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#ifndef FILTER_THROTTLE_H_INCLUDED
+#define FILTER_THROTTLE_H_INCLUDED
+
+#include "filter-expr.h"
+
+FilterExprNode *filter_throttle_new(void);
+
+gboolean filter_throttle_init(FilterExprNode *s, GlobalConfig *cfg);
```

Comment:
With the setter functions this does not need to be public, so you can remove it from the header, and make it static in the `.c` file.